src/test/java/FetchOptionsServletTest.java

```diff
@@ -0,0 +1,26 @@
+import com.google.sps.servlets.FetchOptionsServlet;
+import static org.mockito.Mockito.*;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import java.util.ArrayList;
+import java.util.List;
+
+@RunWith(JUnit4.class)
+public class FetchOptionsServletTest {
+
+  @Test
+  public void titleCaseItemsTest() {
```

Comment:
Please add more tests:
Like pointed out in code, force the first letter to be Caps as well.

Reply:
What do you mean by "Force the first letter to be caps as well?"

Reply:
Fixed!

Reply:
I don't see more tests.

Reply:
I realized that the problem may be that you have only a certain # of commits selected, and my newest commit is not included in your selection. Can you try that and tell me if you see anything new then?